float64 time
int32 mode
---
bool success
